content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,66 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- Adds support for [`cy.session()`](/api/commands/session) to experimental
+  WebKit. Ensures that with `experimentalSessionAndOrigin` enabled, tests are
+  still able to run against webkit. Addresses
+  [#24116](https://github.com/cypress-io/cypress/issues/24116),
+  [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses
+  [#23379](https://github.com/cypress-io/cypress/issues/23379),
+  [#23766](https://github.com/cypress-io/cypress/issues/23766),
+  [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the
+    Cypress Launchpad after a testing type has been selected.
+  - Users will be prompted to record runs to the Dashboard from both the Cypress
+    App and Cypress Launchpad if a project is connected to the Dashboard and has
+    no recorded runs.
+
+**Bugfixes:**
+
+- Fixed issue connecting to the cloud when a self-signed cert was in the cert
```

Comment:
Is this fix still in @mschile?

Reply:
Yes this fix (to revert the change from 10.10.0) is still in